app/templates/response/add_note.js.html

```diff
@@ -16,17 +16,15 @@
 
         // If validation fields are valid, proceed with next click function
         if ($('#add-note-form').parsley().isValid()) {
-            var formData = {
-                request_id: "{{ request.id }}",
-                template_name: "email_response_link.html",
-                type: "note_email"
-            };
-
             $.ajax({
                 url: "/response/email",
                 type: 'POST',
-                data: JSON.stringify(formData),
-                contentType: false,
+                data: {
+                    request_id: "{{ request.id }}",
+                    template_name: "email_response_link.html",
+                    type: "note_email",
+                    privacy: $(".note-privacy:checked").val()
```

Comment:
I recommend using `prop` here, and in other places you are looking for 'checked'